tools/cachemgr.cc

```diff
@@ -693,6 +693,7 @@ read_reply(int s, cachemgr_request * req)
             }
 
             istate = isActions;
+            /*FALLTHROUGH*/
         /* yes, fall through, we do not want to loose the first line */
```

Comment:
I hope this change is unnecessary or, at the very least, can be significantly improved. AFAICT from GCC [documentation](https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html), using `-Wimplicit-fallthrough=2` with GCC would avoid this and similar changes (or, at the very least, make them much nicer than the proposed variant). Since clang [failed](https://reviews.llvm.org/D73852#1901836) to add proper support for such comments, we should rely on GCC and, hence, can use GCC powerful comment support.

The same request applies to other similar changes. <!-- In general, we should take care of this code instead of just brute-forcing our way through compiler warnings. The former approach improves Squid. The latter may actually make it (slightly) worse overall. -->